portfolio/src/main/webapp/index.html

```diff
@@ -174,8 +174,43 @@ <h4>
         <div class="info">     
           <h2>Projects</h2>
 
-          <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-          </p>
+          <br>
+          <br>
+
+          <div class="container">
+            <div class="card">
+              <div class="imgBx">
```

Comment:
Can you put your CSS directly on the img? Generally try to avoid extra divs unless necessary.